setup.cfg

```diff
@@ -1,8 +1,18 @@
 [flake8]
-ignore=E133,E226,E228,N802,N803,N806
-exclude = __init__.py
+ignore=E133, E226, E228, N802, N803, N806, N812, N813, N815, N816, W504
```

Comment:
Please use W503 instead of W504 here and below. LSST has switched (at my request) to support formatting with `black`.